source/_integrations/homewizard.markdown

```diff
@@ -52,3 +52,12 @@ The HomeWizard Energy API only exposes properties that are used within the HomeW
 | Total Gas | m3 | HWE-P1 | Current gas import reading, only available when your smart meter is connected to a gas meter. |
 | DSMR Version | | HWE-P1 | The detected DSMR version. |
 | Smart Meter Model | | HWE-P1 | The detected smart meter model. |
+
+## Switches
+
+The Wifi Energy Socket (`HWE-SKT`) outlet state can be controlled the switch platform. There are two switches:
+
+| Name | Description |
+| --- | --- |
+| Energy Socket | Controls the outlet state of the Energy Socket. |
+| Switch Lock | Forces the outlet state in the `on` position and disables the button. This option is useful when the socket is used for a device that must not be turned off, such as a refrigerator. |
```

Comment:
We could avoid tables here (as they are harder to read/use on mobile), instead this can be a simple list:

- **Energy Socket**: Controls the outlet state of the Energy Socket.

And so on...

Reply:
Removed the table and renamed 'Energy Socket' to 'Switch' (https://github.com/home-assistant/core/pull/64084#discussion_r788386726)